src/usr/local/emhttp/plugins/tailscale/include/Tailscale/System.php

```
@@ -66,6 +66,56 @@
+                Utils::logmsg("Serve TCP Port {$configPort} conflicts with WebGUI, removing");
+                $localAPI->resetServeConfig();
+                Utils::run_command(self::RESTART_COMMAND);
+
+                return;
+            } else {
+                if ((intval(date("i")) % 10 == 0)) {
+                    Utils::logmsg("Checked for WebGUI conflict with serve TCP Port {$configPort}");
+                }
+            }
+        }
+    }
```

PHPStan (2.0.3)

Syntax error, unexpected '}', expecting EOF on line 117


(phpstan.parse)